### PR TITLE
HParams: Allow an input to force use of legacy table

### PR DESCRIPTION
**Pull request by JamesHollyer**
## Motivation for features / changes
We will soon be flipping the flag to enable the RunsDataTable. However, this new table does not work on the hosted experiments list page. This allows that page(which is internal only) to force the runs table to use the old table so we do not break that feature.

## Detailed steps to verify changes work correctly (as executed by you)
I ran this code internally first and used it to fix the problem. Then copied it out to OSS.

### CODE REVIEW

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts [hunk 1]
```diff
@@ -816,6 +817,11 @@ export class RunsTableContainer implements OnInit, OnDestroy {
   orderColumns(newHeaderOrder: ColumnHeader[]) {
     this.store.dispatch(runsTableHeaderOrderChanged({newHeaderOrder}));
   }
+
+  useDataTable() {
+    console.log('forceLegacyTable', this.forceLegacyTable);
```
Comment (bmd3k) [inline tensorboard/webapp/runs/views/runs_table/runs_table_container.ts:822]: Can you remove this console.log?

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts [hunk 2]
```diff
@@ -816,6 +817,11 @@ export class RunsTableContainer implements OnInit, OnDestroy {
...
+    return this.HParamsEnabled.value && !this.forceLegacyTable;
```
Comment (bmd3k) [inline tensorboard/webapp/runs/views/runs_table/runs_table_container.ts:823]: Can you rename HParamsEnabled to hparamsEnabled or hParamsEnabled?

**Review by bmd3k — APPROVED** [review]

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts (hunk 1 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_table_container.ts]: Doh! Removed

File: tensorboard/webapp/runs/views/runs_table/runs_table_container.ts (hunk 2 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_table_container.ts]: Changed.

**Pull request merged**